src/main/java/uk/gov/justice/hmpps/prison/api/model/AgyPrisonerPayProfile.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import lombok.Builder;
+import lombok.Data;
+
+@Schema(description = "Agency prisoner pay profile")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder(toBuilder = true)
+@Data
+public class AgyPrisonerPayProfile {
+
+  @Schema(description = "Agency identifier", example = "MDI")
+  private String agencyId;
+
+  @Schema(description = "The start date when this pay profile took effect", example = "2022-10-01")
+  private LocalDate startDate;
+
+  @Schema(description = "The end date when this pay profile will stop taking effect", example = "2027-10-01")
+  private LocalDate endDate;
+
+  @Schema(description = "Whether automatic payments are enabled", example = "Y")
+  private String autoPayFlag;
```

Comment:
is it best to map this to a boolean instead?

Reply:
Yes, I think you're right. On it.

Reply:
Converted to a boolean for the model object, from the Y/N flag in the entity.